portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,10 +54,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      num_cmnts++;
-      String Cmt = request.getParameter("cmnt");
-      comments.add("#Comment"+num_cmnts+": "+Cmt);
-      response.sendRedirect("/index.html");
+    num_cmnts++;
```

Comment:
Nit: Local variable names should be written in lowerCamelCase (https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names)

Reply:
fixed the issue